caput/memh5.py

```diff
@@ -118,6 +118,18 @@ def __eq__(self, other):
         return Mapping.__eq__(self, other) and self._dict == other._dict
 
 
+class rw_dict(ro_dict):
```

Comment:
This isn't really necessary. It's in there to mimic the appearance of other properties which return `ro_dict` objects, for consistency